build/configs/rtl8721csm/tc/defconfig

```diff
@@ -136,15 +137,24 @@ CONFIG_BOARD_LOOPSPERMSEC=16717
 #
 # Interrupt options
 #
-CONFIG_ARCH_HAVE_INTERRUPTSTACK=y
-CONFIG_ARCH_INTERRUPTSTACK=0
+# CONFIG_ARCH_HAVE_INTERRUPTSTACK is not set
 # CONFIG_ARCH_HAVE_NESTED_INTERRUPT is not set
-CONFIG_ARCH_HAVE_HIPRI_INTERRUPT=y
-# CONFIG_ARCH_HIPRI_INTERRUPT is not set
+# CONFIG_ARCH_HAVE_HIPRI_INTERRUPT is not set
+
+#
+# Boot Memory Configuration
+#
+CONFIG_RAM_REGIONx_START="0x0,0x0,"
```

Comment:
Please set the proper physical RAM address.

Reply:
We cannot hard-code this address in defconfig, this is determined by the size of the application and after the linker has successfully run. The start address and size is initialized in os_heap_init in rtl8721dhp_app_start.c